packages/snap-url-manager/src/Translators/Url/UrlTranslator.ts

```diff
@@ -179,11 +179,17 @@ export class UrlTranslator implements Translator {
 				} else if (decodedHashEntries.length && decodedHashEntries.length <= 3) {
```

Comment:
Is it possible to get rid of this condition and add additional logic in the <=4 block?